code/__DEFINES/footstep.dm

```diff
@@ -136,13 +136,13 @@ GLOBAL_LIST_INIT(clawfootstep, list(
 		'sound/effects/footstep/woodclaw2.ogg',
 		'sound/effects/footstep/woodclaw3.ogg',
 		'sound/effects/footstep/woodclaw2.ogg',
-		'sound/effects/footstep/woodclaw1.ogg'), 90, 1),
+		'sound/effects/footstep/woodclaw1.ogg'), 70, 1),
 	FOOTSTEP_HARD_CLAW = list(list(
 		'sound/effects/footstep/hardclaw1.ogg',
 		'sound/effects/footstep/hardclaw2.ogg',
 		'sound/effects/footstep/hardclaw3.ogg',
 		'sound/effects/footstep/hardclaw4.ogg',
-		'sound/effects/footstep/hardclaw1.ogg'), 90, 1),
+		'sound/effects/footstep/hardclaw1.ogg'), 75, 1),
```

Comment:
Still trying to lower Vox walk sounds, I see.

Reply:
As I said in the PR's description, I lowered the volume of the claw footstep sounds because they were obnoxiously loud otherwise when I was testing. I'll admit to having been inspired by Carthusia's [https://github.com/ParadiseSS13/Paradise/pull/15649#issuecomment-788957383](comment), but other than that this is in no way vox related.
It lowers the footstep volume for Unathi, Tajaran, Kidan and Vulpkanin to the same level.

Reply:
Back at my computer again, so just to give reference here's the amplitude of the different sound files: (For wood flooring)
![Amplitude](https://user-images.githubusercontent.com/57483089/125055015-b7eab900-e09e-11eb-8e50-8d82c808ef38.PNG)
![image](https://user-images.githubusercontent.com/57483089/125055701-65f66300-e09f-11eb-8510-810f3f04d598.png)
The `woodclaw.ogg` sounds are a full 5dB louder than the the highest from the others. -2.3dB vs -7.3dB.
It would probably make more sense to normalise them to be equal with the rest of the sounds, but I figured it would be best to leave the actual files as-is and just modify the volume in the code. I could easily edit the files instead though if that's requested.